doc/CLI.rst

```diff
@@ -44,7 +44,7 @@ Pythran can also generate raw C++ code, using the ``-e`` switch::
 
   $> pythran -e cli_foo.py -o cli_foo.hpp
   $> printf '#include \"cli_foo.hpp\"\nusing namespace __pythran_cli_foo ; int main() { foo()(); return 0 ; }' > cli_foo.cpp
-  $> `pythran-config --compiler` cli_foo.cpp `pythran-config --cflags --libs` -o cli_foo
+  $> `pythran-config --compiler` -std=c++11 cli_foo.cpp `pythran-config --cflags --libs` -o cli_foo
```

Comment:
I removed it because I don't want to force the mode. What if the guy
wants std=c++14, or if its compiler uses different flag for c++11
support (like windows) --cflags only outputs -D, -I

That's the way pkg-config works, at least